index.html

```diff
@@ -278,22 +280,269 @@
             </div>
 
             <footer class="floe-footer">
-                <div class="floe-content row">
+                <div class="floe-content row align-center">
                     <div class="columns large-6 small-12">
-                        <div>
-                            <p>
-                                Floe is a project of the
-                                <a href="http://idrc.ocadu.ca">Inclusive Design Research Centre</a>, <a href="http://ocadu.ca">OCAD University</a>.<br/>
-                            </p>
-                            <p>
-                                <span class="floe-hewlett-logo">Funded by a grant from <a href="http://www.hewlett.org">The William and Flora Hewlett Foundation</a>.</span>
-                            </p>
-                        </div>
+                            <div class="row align-center idrc-credit-row">
+                                <div class="idrc-logo-container columns large-5 medium-7 small-9">
+                                    <title>IDRC Logo</title>
```

Comment:
Ran through the validator and fixed an ID duplication issue on the SVGs. There were aslo some markup complaints with the UIO markup on the page due to duplicate IDs between the smallScreen and wideScreen markup. I've updated the ids to be distinct without any noticeable impact, but want to note this - I don't think the IDs have any implications because all events and DOM binding should be based on the classes, correct?